index.d.ts

```diff
@@ -1,9 +1,12 @@
 declare module 'coordinate-parser' {
-    class Coordinates {
-        constructor(coordinateString: string)
-        getLatitude(): number
-        getLongitude(): number
-    }
+  class CoordinateParser {
+    constructor(coordinateString: string);
+    getLatitude(): number;
+    getLongitude(): number;
+  }
 
-    export default Coordinates
+  module CoordinateParser {}
+
+  export = CoordinateParser;
```

Comment:
And this line is equal to 
```
export default CoordinateParser
```

Reply:
Not quite. The equals sign makes the difference between having jgeiger's error or not having it.